src/applications/vaos/appointment-list/components/ConfirmedAppointmentDetailsPage/index.jsx

```diff
@@ -69,28 +73,51 @@ export default function ConfirmedAppointmentDetailsPage() {
 
   useEffect(
     () => {
-      let pageTitle;
+      let pageTitle = 'VA appointment on';
       let prefix = null;
 
-      if (isPast) prefix = 'Past';
-      else if (isCanceled) prefix = 'Canceled';
+      if (selectIsPast(appointment)) prefix = 'Past';
+      else if (selectIsCanceled(appointment)) prefix = 'Canceled';
 
       if (isCommunityCare)
-        pageTitle = prefix ? `${prefix} community care` : 'Community care';
+        pageTitle = prefix
+          ? `${prefix} community care appointment on`
+          : 'Community care appointment on';
       else if (isInPerson) {
         if (appointment?.vaos?.isCompAndPenAppointment)
-          pageTitle = prefix ? `${prefix} claim exam` : 'Claim exam';
-        else pageTitle = prefix ? `${prefix} in-person` : 'In-person';
-      } else if (isVideo) pageTitle = prefix ? `${prefix} video` : 'Video';
-      else if (isVAPhoneAppointment) {
-        pageTitle = prefix ? `${prefix} phone` : 'Phone';
+          pageTitle = prefix
+            ? `${prefix} claim exam appointment on`
+            : 'Claim exam appointment on';
+        else
+          pageTitle = prefix
+            ? `${prefix} in-person appointment on`
+            : 'In-person appointment on';
+      }
+      if (isVideo) {
+        pageTitle = prefix
+          ? `${prefix} video appointment on`
+          : 'Video appointment on';
+        if (isClinicVideoAppointment(appointment)) {
+          pageTitle = prefix
+            ? `${prefix} video appointment at VA location on`
```

Comment:
I believe the spec says `{prefix} video appointment at a VA location on` and this is missing the `a`.

Reply:
Good catch! Thanks.

Reply:
Actually, it states:

{prefix} video appointment at VA location on [Weekday], [Month] [DD], [YYYY] | Veterans Affairs

Reply:
Ah oops, I was reading the line above it:
<img width="487" alt="image" src="https://github.com/user-attachments/assets/cb4d01f4-ca0c-4388-9386-206d94ecc00d">
But this is odd, I think it reads better with an `a` and the ATLAS appointments have `an` in both of the parts in the checklist. Not sure why there's a discrepancy for the "at VA location" appointments. Can we check with design on this?

Reply:
Good catch, please do include the "a" - `Upcoming video at a VA location` - thanks!

Reply:
Done.